### PR TITLE
Fix print function with tf_logging.info to keep consistence

### PR DESCRIPTION
**Pull request by imsheridan**
This PR is to fix print function with tf_logging.info to keep consistence.

Within the single file, some places uses print() while some places use tf_logging.info.

### CODE REVIEW

**Review by jhseu — DISMISSED** [review]

**Comment by jhseu** [comment]
Mind addressing the lint errors? (Line too long)

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jhseu: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jhseu: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Review by jhseu — APPROVED** [review]

File: tensorflow/python/kernel_tests/pooling_ops_test.py [hunk 1]
```diff
@@ -947,7 +947,7 @@ def _ConstructAndTestGradient(self,
           output_sizes,
           x_init_value=x_init_value,
           delta=1e-2)
-    print("%s gradient error = " % func_name, err)
+    tf_logging.info("%s gradient error = " % func_name, err)
```
Comment (jerryyin) [inline tensorflow/python/kernel_tests/pooling_ops_test.py]: This is broken, should be:
 `tf_logging.info("%s gradient error = %.4f" % (func_name, err))`

File: tensorflow/python/kernel_tests/pooling_ops_test.py [hunk 2]
```diff
@@ -1024,7 +1024,7 @@ def _ConstructAndTestSecondGradient(self,
           input_sizes,
           x_init_value=x_init_value,
           delta=1e-2)
-    print("%s second-order gradient error = " % func_name, err)
+    tf_logging.info("%s second-order gradient error = " % func_name, err)
```
Comment (jerryyin) [inline tensorflow/python/kernel_tests/pooling_ops_test.py]: Should be:
`tf_logging.info("%s second-order gradient error = %.4f" % (func_name, err))`

**Pull request merged**